sunkit_pyvista/plotter.py

```diff
@@ -417,19 +417,19 @@ def plot_quadrangle(
 
     def plot_field_lines(self, field_lines, *, color_func=None, **kwargs):
         """
-        Plot magnetic field lines from `pfsspy`.
+        Plot magnetic field lines.
 
         Parameters
         ----------
-        field_lines : `pfsspy.fieldline.FieldLines`
+        field_lines : `sunkit_magex.pfss.fieldline.FieldLines`
```

Comment:
Does this not need to be added to the docs config?

We should also enable nitpicky

Reply:
yes, I need to add it to intersphinx. I am not doing template or infra in this PR. I am on a fast path to working lol

Reply:
I can't seem to make intersphinx work

Reply:
I can do it after this PR.